bodhi/server/templates/update.html

```diff
@@ -508,6 +542,31 @@ <h1>
             % else:
               <a id='revoke' class="btn btn-sm btn-danger"><span class="fa fa-fw fa-arrow-circle-left"></span> Revoke</a>
             % endif
+            % if request.registry.settings.get('test_gating.required') and not update.test_gating_passed:
+              <a id='waive' class="btn btn-sm btn-primary" data-toggle="modal" data-target="#waive_test_results">Waive Test Results</a>
+              <div class="modal fade" id="waive_test_results" tabindex="-1" role="dialog" aria-labelledby="waiveModalLabel" aria-hidden="true">
```

Comment:
Can this ```<div>``` be moved to be somewhere else in the code? It's a lot of code and it makes it a little hard to read the already complicated if/else block that it's inside.

Reply:
Sure, I guess I could move it down to the bottom.